src/handbook/development/releases/process.md

```diff
@@ -3,85 +3,161 @@ navTitle: Release Process
 ---
 # Release Process
 
-There are two processes, one for major/minor point releases. The second for patch
-releases.
+We do two types or releases:
 
-## Major/minor point releases
+ - Scheduled releases at a regular [monthly cadence](./planning.md#cadence). This
+   encompasses the entire set of product repositories.
+ - Fix releases of individual repositories as needed
+
+## Scheduled Releases
+
+We publish a new release of FlowForge [every four weeks](./planning.md#cadence). Each
+release has a Release Manager who oversees the process. This role rotates around
+the team to ensure everyone is able to do it and that we don't have any single
+points of failure in the process.
 
 ### Setup
 
- - Decide who will be Release Manager for this release. For Major/Minor releases this should be shared across the whole team to prevent it becoming a single point of failure. For Fix releases it can be the developer committing the fix.
- - Create a Release checklist issue using the [Release Checklist template](https://github.com/flowforge/admin/issues/new?assignees=&labels=&template=release.md&title=Release%3A){rel="nofollow"}
+The Release Manager should take the following steps at the *start* of the release
+iteration:
+
+  - Create a Release checklist issue using the [Release Checklist template](https://github.com/flowforge/admin/issues/new?assignees=&labels=&template=release.md&title=Release%3A){rel="nofollow"}.
  - Assign the issue to the Release Manager
- - Ensure you have the following installed on your machine:
-    - git command-line tools.
+ - Ensure the following tools are installed and working:
+    - git command-line tools
+      - Ensure ssh key access is setup so git interactions do not require continual
+        password typing - `ssh -T git@github.com`
     - [GitHub client](https://github.com/cli/cli)
+      - Ensure the client is logged in using `gh auth login`
     - [jq](https://stedolan.github.io/jq/download/)
     - [yq](https://mikefarah.gitbook.io/yq/#install)
- - Ensure you're machine is authenticated with the GitHub client: `gh auth login`
- - Ensure you are able to access the FlowForge repositories using the git cli without being prompted for a password: `ssh -T git@github.com`
- - When we are ready to start the release process the Release Manager should start a Slack Huddle in #dev and keep that open until the release is completed.
 
-### Unmanaged Repositories
+On release day the Release Manager takes the lead on running the release process,
+following the steps in the checklist.
```

Comment:
```suggestion
On release day, the Release Manager takes the lead in running the release process, following the steps in the checklist.
```

I propose a comma after "On release day" to separate the introductory clause from the main clause